.github/workflows/ci.yml

```diff
@@ -587,9 +587,12 @@ jobs:
       - environment=${{ needs.setup.outputs.runner-env }}
       - cpu
       - os-family=Linux
+    strategy:
+      matrix:
+        architecture: [rv64, rv32-linux]
```

Comment:
I feel like we can put the tests for all platforms and architectures (android, x86, ...) into this matrix.

Can you also include docker image and platform as the parameters? (like https://github.com/iree-org/iree/blob/main/.github/workflows/benchmarks.yml#L93) So later we can put everything into the matrix.

Reply:
Can we please push it to a later PR? This improvement may be out of scope for the original PR.